src/test/javascript/portal/filter/FilterGroupPanelSpec.js

```diff
@@ -80,65 +75,53 @@ describe("Portal.filter.FilterGroupPanel", function() {
     });
 
     describe('filter sorting', function() {
-        var expectedReturn;
-        var filterConfigs;
-        var testBooleanFilterA;
-        var testDateFilter;
-        var testDateRangeFilter;
-        var testBooleanFilterE;
-        var testBboxFilter;
-        var testStringFilter;
 
-        beforeEach(function() {
-            layer = {
-                server: {
-                    uri: {}
-                }
-            };
+        it('sorts panels in expected order', function() {
 
-            cfg = {
-                layer: layer
-            }
+            spyOn(Portal.filter.BooleanFilterPanel.prototype, '_createField');
+            spyOn(Portal.filter.BooleanFilterPanel.prototype, '_setExistingFilters');
+            spyOn(Portal.filter.NumberFilterPanel.prototype, '_setExistingFilters');
+            spyOn(Portal.filter.NumberFilterPanel.prototype, 'setLayerAndFilter');
+            spyOn(Portal.filter.GeometryFilterPanel.prototype, '_createField');
+            spyOn(Portal.filter.GeometryFilterPanel.prototype, 'setLayerAndFilter');
+            spyOn(Portal.filter.DateFilterPanel.prototype, '_setExistingFilters');
+            spyOn(Portal.filter.DateFilterPanel.prototype, '_createField');
+            spyOn(Portal.filter.ComboFilterPanel.prototype, '_setExistingFilters');
```

Comment:
A number of these calls to `spyOn()` will hopefully be unnecessary after I've done some of the refactoring we have planned.